src/audio/volume/volume.h

```diff
@@ -6,7 +6,7 @@
  */
 
 /**
- * \file audio/volume.h
+ * \file volume/volume.h
```

Comment:
line 9 change is because now header is located in src/audio/volume/volume.h, and normally, src/audio will be set to include path, so volume/volume.h just means path.

__SOF_AUDIO_VOLUME_H__ is quite good, I have no other good names to replace, :)

Reply:
lets fix this incrementally.

Reply:
@btian1 `SOF_AUDIO_VOLUME_H` means that the header is located under sof/audio/volume.h. After you've moved it is could be something like `AUDIO_VOLUME_VOLUME_H` but you'd need to check what others use